diff --git a/build_files/build_environment/install_deps.sh b/build_files/build_environment/install_deps.sh
index 5cf005d60ce..7017a2e7ea2 100755
--- a/build_files/build_environment/install_deps.sh
+++ b/build_files/build_environment/install_deps.sh
@@ -1124,7 +1124,7 @@ compile_Python() {
 
     ./configure --prefix=$_inst --libdir=$_inst/lib --enable-ipv6 \
         --enable-loadable-sqlite-extensions --with-dbmliborder=bdb \
-        --with-computed-gotos --with-pymalloc
+        --with-computed-gotos --with-pymalloc CFLAGS='-fPIC' CPPFLAGS='-fPIC'
 
     make -j$THREADS && make install
     make clean
